documentation/user-interface/fixture-editor.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: docs
+title: Fixture Editor
+permalink: /documentation/user-interface/fixture-editor.html
+---
+# Fixture Editor
+
+The **Fixture** editor allows you to view and modify a fixture. The editor opens when you double-click the fixture in the [Explorer](explorer-panel.md) panel.
+
+The editor consists of the following parts:
+
+* The toolbar -Provides information about the fixture (the name and tested webpage's URL) and buttons for performing [typical tasks](#typical-tasks) with the fixture.
+
+    The webpage's URL is clickable and leads to the webpage opened in the browser.
+
+* The test list - Displays tests included to the fixture. The tests' rows are clickable and lead to the [Test](test-editor.md) editor.
```

Comment:
`The tests' rows are clickable and lead to the [Test](test-editor.md) editor.`

Again. Not just complex but a little robotic.